### DISCUSSION TITLE
[FR] Option to select ALL assets from a volume

### DISCUSSION THREAD

**Discussion by thupsi** [craftcms/cms#11311] [Ideas]
Currently when the user navigates to the root folder of an asset volume and checks ```select all``` only the visible assets (e.g. the first 100 assets from the root folder) are selected. I would like to have the ability to select **all assets from the current folder** and **all assets from the current folder including subfolders**. 

From a UI point of view when checking ```select all``` there could be a notice like gmail:

<img width="450" alt="image" src="https://user-images.githubusercontent.com/19857599/170196423-8b892618-e091-4061-9820-75ea0e0bdf07.png"> 

…with the addition of an option to include subfolders.

My use case for this is to easily download all assets when there are many subfolders, but I'm sure there are additional cases with plugins to bulk resize etc.

**Comment by brandonkelly** [comment]
This was requested previously by #10295. It’s on our list, but needs to happen alongside addressing #8794.
